core/test/audits/mainthread-work-breakdown-test.js

```diff
@@ -91,47 +119,57 @@ Object {
     expect(Math.round(output.numericValue)).toMatchInlineSnapshot(`979`);
     assert.equal(output.details.items.length, 7);
     assert.equal(output.score, 1);
-  });
-
-  it('should compute the correct values for the load trace (legacy)', async () => {
```

Comment:
@connorjclark is there a reason we need to keep this around since it wasn't removed in https://github.com/GoogleChrome/lighthouse/issues/15005?

Reply:
Yes, as long as we still have code handling [`TracingStartedInPage`](https://github.com/GoogleChrome/lighthouse/blob/2a215be64748c4dfc4d3961ed53b8a6e725d03d5/core/lib/tracehouse/trace-processor.js#L444) for back compat.

Reply:
Ok. I reintroduced this test case, however the old trace we have stored causes a [`NO_DCL`](https://github.com/GoogleChrome/lighthouse/blob/main/core/computed/metrics/interactive.js#L159) error when we try and compute the TBT. For this reason, I've changed the error handling logic to bypass all errors when computing TBT and report to sentry.